lib/std/math/big/int_test.zig

```diff
@@ -286,6 +286,30 @@ test "big.int string set bad base error" {
     try testing.expectError(error.InvalidBase, a.setString(45, "10"));
 }
 
+test "big.int string set empty string" {
+    var a = try Managed.init(testing.allocator);
+    defer a.deinit();
+
+    try a.setString(10, "");
+    try testing.expect(a.eqZero());
+}
+
+test "big.int string set underscores" {
+    var a = try Managed.init(testing.allocator);
+    defer a.deinit();
+
+    try a.setString(10, "__________");
+    try testing.expect(a.eqZero());
+}
+
+test "big.int string set base 36" {
```

Comment:
We are doing base36 decoding here, so adding a hint for it `(base36 decoding)` sounds like a good idea. Does the base36 encoding into bigint also work?

Reply:
yes it does. in fact, the code was already able to encode and decode base36 numbers, the only thing to change were the `assert` that checked the base.